wolfcrypt/src/rsa.c

```diff
@@ -299,12 +308,20 @@ int wc_InitRsaKey_ex(RsaKey* key, void* heap, int devId)
     #endif
 
     #ifdef WC_ASYNC_ENABLE_RSA
-        /* handle as async */
-        ret = wolfAsync_DevCtxInit(&key->asyncDev, WOLFSSL_ASYNC_MARKER_RSA,
-                                                            key->heap, devId);
-        if (ret != 0)
-            return ret;
+        #if defined(HAVE_PKCS11)
+            if (!isPkcs11) {
```

Comment:
Move braces outside the macro. It should just wrap the if

Reply:
Done